src/main/java/org/fcrepo/camel/FcrepoClient.java

```diff
@@ -81,7 +82,7 @@ public FcrepoResponse head(final URI url)
         final HttpRequestBase request = HttpMethods.HEAD.createRequest(url);
         final HttpResponse response = executeRequest(request);
         final int status = response.getStatusLine().getStatusCode();
-        final String contentType = getContentTypeHeader(response);
+        final Optional<String> maybeContentType = getContentTypeHeader(response);
```

Comment:
Well, so this is the thing. Here're some thoughts:

Some of the gains in brevity in the main code are lost by expanded test code. You're eventually losing a few dozen lines over a few hundred lines, so the real question is: is it clearer? That's a matter of taste. See below for further comment.